src/Arcus.Templates.Tests.Integration/Worker/Fixture/TestOrderAzureServiceBusMessageHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
```

Comment:
What exactly are we testing here ?
You're testing if the generated project based on the template works as expected ? 

I was more thinking: maybe we should test if the generated files are as expected?  But this might actually be a good approach as well.

Reply:
Yeah, the integration tests in the templates library all test the actual workings. So, if a setting of a project option results in an extra file, it checks the general workings instead of just that file. Seems a good approach to test the resulted project as a whole, no? So, in this case, we test indeed if we can receive an Azure Service Bus message via our generated project from the template. That way, we're sure that the template works.

Reply:
I agree.  This is the best approach.